### DISCUSSION TITLE
Can not find dependency when using DBeaver (在使用DBeaver连接IoTDB时无法找到对应的Maven依赖)

### DISCUSSION THREAD

**Discussion by Theyoungwolf1201** [apache/iotdb#5650] [Q&A] [answered]
您好，在尝试使用DBeaver连接IoTDB时发现无法找到官网教程中提到的Maven依赖，可以麻烦分享一下Maven依赖或在教程中更新一下吗？十分感谢！

**Comment by HTHou** [comment, marked as answer]
可以下载0.13.0版本的源代码https://github.com/apache/iotdb/releases/download/v0.13.0/apache-iotdb-0.13.0-source-release.zip ，然后解压执行`mvn clean package -pl jdbc -am -DskipTests -P get-jar-with-dependencies` 后，在jdbc/target/ 下找到并添加名为iotdb-jdbc-0.13.0-jar-with-dependencies.jar的包。 只需要添加这一个包就行。官网的图有点没说清楚，我们之后更新一下！

You can download the source code (https://github.com/apache/iotdb/releases/download/v0.13.0/apache-iotdb-0.13.0-source-release.zip), unzip,  and run `mvn clean package -pl jdbc -am -DskipTests -P get-jar-with-dependencies`. Then you can find iotdb-jdbc-0.13.0-jar-with-dependencies.jar under folder `jdbc/target`

**Reply by Theyoungwolf1201** [reply]
十分感谢您的帮助
:)
